spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```

Comment:
- [ ] In `posts_spec.rb`, there are 4 tests which are failing. kindly fix this issue
![image](https://user-images.githubusercontent.com/81932478/187309811-08db1cbb-19dc-4306-8449-b35ae0aa47c9.png)